bal/p2p.py

```diff
@@ -33,18 +38,28 @@ def transaction_pool(self):
 
     def query(self, peer_addr, message):
         threading.Thread(
-                target = self.send_message,
-                args = (peer_addr, message)
+            target=self.send_message,
+            args=(peer_addr, message)
         ).start()
 
     def add_peer(self, peer_str):
         peer_addr = self.get_peer_tuple(peer_str)
         if peer_str in self.peer_sockets.values():
             return False
         else:
-            self.query(peer_addr, Message(MessageType.QUERY_LATEST_BLOCK, '', self.p2p_addr))
+            self.query(
```

Comment:
these are seems ok without splittin, I mean they're not that too long. What do you say? Maybe we can keep them as they're?